10/collection-meta.yaml

```
@@ -301,6 +301,12 @@ collections:
       - tadeboro
       - mancabizjak
     repository: https://github.com/sensu/sensu-go-ansible
+    removal:
+      major_version: 12
+      announce_version: 10.5.0
+      reason: guidelines-violation
+      discussion: https://forum.ansible.com/t/8380
+      reason_text: https://github.com/sensu/sensu-go-ansible/issues/362
```

reason_text should not be an URL, but a text (using Ansible markup). This is added as-is into the changelog fragment and the docsite.
```suggestion
      reason_text: There have been L(unresolved Ansible inclusion requirement violations, https://github.com/sensu/sensu-go-ansible/issues/362).
```

---

Then the final entry generated by https://github.com/ansible-community/antsibull/pull/623 would like like

> The {collection} collection will be removed from Ansible {removal.major_version} due to violations of the Ansible inclusion requirements. *There have been L(unresolved Ansible inclusion requirement violations, https://github.com/sensu/sensu-go-ansible/issues/362).*

which seems redundant. I think there should just be a field to include a link to the collection issue and then `violations of Ansible inclusion requirements` in the first sentence generated by antsibull should be a hyperlink to that issue.

---

The policy for collection requirements violations always requires filing an issue with the collection, so I think having a dedicated field for that issue is reasonable.

---

That needs an extension of the schema. I'm not sure that issue actually needs to be linked here though, since it should already be linked from the discussion, which is linked here.

---

> Then the final entry generated by [ansible-community/antsibull#623](https://github.com/ansible-community/antsibull/pull/623) would like like
> 
> > The {collection} collection will be removed from Ansible {removal.major_version} due to violations of the Ansible inclusion requirements. _There have been L(unresolved Ansible inclusion requirement violations, [sensu/sensu-go-ansible#362](https://github.com/sensu/sensu-go-ansible/issues/362))._
> 
> which seems redundant. I think there should just be a field to include a link to the collection issue and then `violations of Ansible inclusion requirements` in the first sentence generated by antsibull should be a hyperlink to that issue.

Since there can be zero, one, or multiple links to issues in the repo, and possible text or not text with other explanations, I think it's best to simply link to the discussion and have all further links there, and not in the changelog / docsite.

The text here should be a short recap of what the violations are.

---

How about this?

```suggestion
      reason_text: >
        The collection has
        L(unresolved sanity test failures, https://github.com/sensu/sensu-go-ansible/issues/362).
```

---

Sounds good to me!